README.md

```diff
@@ -86,6 +86,35 @@ Computacenter TechSource - this app will post cap update requests to TechSource
 
 The app should be available at https://get-help-with-tech-(dev|staging|prod).london.cloudapps.digital
 
+### If you need to rollback a release
+
+Sometimes releases fail on production - sometimes this is due to a migration which fails due to unexpected data in production, for instance.
+When this happens, you'll probably want to minimize user impact by rolling back the release to a previous version ASAP, while you investigate offline.
+
+There are some make tasks that can help:
+
+`make (env) remote-docker-tags` - list all available tags for the given environment's Docker image, on Docker Hub
+`make (env) rollback-to TAG=...` - re-deploy the Docker image tagged with the given TAG
```

Comment:
I appreciate that you're trying to be generic here, but the reality is this command is likely to be only ever needed on prod, so I'd just put that in to make it easier to copy and paste the command:

```suggestion
`make prod remote-docker-tags` - list all available tags for production Docker image, on Docker Hub (you can replace `prod` with another environment)
`make prod rollback-to TAG=...` - re-deploy the Docker image tagged with the given TAG
```